src/libstd/path.rs

```diff
@@ -2248,12 +2269,23 @@ impl Path {
     /// This function will traverse symbolic links to query information about the
     /// destination file. In case of broken symbolic links this will return `false`.
     ///
+    /// If you cannot access the directory containing the file, e.g. because of a
+    /// permission error, this will return `false`.
+    ///
     /// # Examples
     ///
     /// ```no_run
     /// use std::path::Path;
     /// assert_eq!(Path::new("./is_a_directory/").is_dir(), true);
     /// assert_eq!(Path::new("a_file.txt").is_dir(), false);
+    /// # See Also
```

Comment:
you need a triple backtick above this line, it's trying to execute

   # See Also

as Rust code. the `///` has nothing to do with it, really.

A `///` below that backtick and above the `/// # See Also` line would be good as well.